policyengine_it/variables/private/mario_besusso/mario_besusso_eligible.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_it.model_api import *
+
+
+class mario_besusso_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for Mario Besusso Fund"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        mb_eligible_groups = parameters(
+            period
+        ).private.mario_besusso.eligibility
+
+        employment_category = person(
+            "employment_category", period
+        ).decode_to_str()[0]
+
+        is_eligible = np.where(
+            employment_category in mb_eligible_groups, True, False
+        )
+
+        return is_eligible
```

Comment:
I would write it as follows:
         employment_category = person(
            "employment_category", period
        )
        categories =  employment_category == employment_category.possible_values
        return employment_category == categories.EXECUTIVE